res/database/Default/database_programmes_fictional.xml

```diff
@@ -1027,10 +1171,12 @@ Exciting entertainment and brain-teasing trivia questions.</en>
 			<title>
 				<de>Seniorensause</de>
 				<en>Seniors Big Bash</en>
+				<pl>Senioreza - O seniorach</pl>
```

Comment:
Originally title means something like "Big Party of Seniors".

Reply:
"Big Party of Seniors" - "Wielka impr**eza** **senior**ów" - Senioreza - compound of two words.
Can stay or change to "Wielka impreza seniorów" ?

Reply:
changed to Wielka impreza seniorów